doc/source/ray-job-submission/overview.rst

```diff
@@ -0,0 +1,239 @@
+
+
+=========================
+From laptop to production
+=========================
+
+.. warning::
+
+    Ray Job Submission is at Alpha phase with APIs mostly stable but subject to change in the future.
+
+Concepts
+--------
+
+- **Package**: A collection of files and configurations that defines an application, thus allowing it to be executed in a different environment remotely (ideally self-contained).
+
+- **Job**: A Ray application that will be submitted to a Ray cluster for execution. Once a job is submitted, it runs once on the cluster to completion or failure. Retries or different runs with different parameters should be handled by the submitter. Jobs are scoped to the lifetime of a ray cluster.
+
+- **Job Manager**: An entity that manages the lifecycle of a Job and potentially multiple ray clusters, such as scheduling, killing, polling status, getting logs, and persisting inputs / outputs. Should be highly available (HA) by default. Can be any framework with these abilities, such as Airflow.
+
+Within the context of job submission, the packaging part is equivalent to :ref:`Runtime Environments<runtime-environments>`, where we can dynamically configure your desirable ray cluster, actor or task level runtime environment for your submitted job.
```

Comment:
Is it accurate to say "the packaging is handled by Runtime Environments"?  "equivalent" might be confusing

Reply:
```suggestion
Within the context of job submission, the packaging part is equivalent to :ref:`Runtime Environments<runtime-environments>`, where we can dynamically configure your desired ray cluster, actor or task level runtime environment for your submitted job.
```
We don't have a notion of cluster-level runtime environment, is this referring to a cluster environment or something else?

Reply:
yes i meant cluster environment, the one we use for `ray.init()`